File: app/src/main/java/com/BlizzardArmory/starcraft/Profile/Snapshot.java
package com.BlizzardArmory.starcraft.Profile;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Snapshot {

    @SerializedName("seasonSnapshot")
    @Expose
    private SeasonSnapshot seasonSnapshot;
    @SerializedName("totalRankedSeasonGamesPlayed")
    @Expose
    private int totalRankedSeasonGamesPlayed;

    public SeasonSnapshot getSeasonSnapshot() {
        return seasonSnapshot;
    }

    public void setSeasonSnapshot(SeasonSnapshot seasonSnapshot) {
        this.seasonSnapshot = seasonSnapshot;
    }

    public int getTotalRankedSeasonGamesPlayed() {
        return totalRankedSeasonGamesPlayed;
    }

    public void setTotalRankedSeasonGamesPlayed(int totalRankedSeasonGamesPlayed) {
        this.totalRankedSeasonGamesPlayed = totalRankedSeasonGamesPlayed;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/starcraft/profile/Snapshot.kt
package com.BlizzardArmory.model.starcraft.profile

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Snapshot.
 */
@Keep
data class Snapshot(

    @SerializedName("seasonSnapshot")
    var seasonSnapshot: SeasonSnapshot,

    @SerializedName("totalRankedSeasonGamesPlayed")
    var totalRankedSeasonGamesPlayed: Int

)